hw14/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body><div class="tasks">
+    <div class="task">
+        <p>Please  enter the year</p>
+        <input type="text" class="input1">
```

Comment:
Lets follow a11y convention here and below
```suggestion
         <label for="input1">Please  enter the year</label>
        <input type="text" class="input1" id="input1">
```